card_test/venv/Scripts/code/BLOB.py

```diff
@@ -0,0 +1,119 @@
+from PIL import Image, ImageDraw
+import numpy as np
+import sys  # We need this in order to change the max recursion
+
+
+def main():
+    # Python normally only allows a function to call itself 10^4 times, which is not enough for larger BLOBs
+    sys.setrecursionlimit(10 ** 6)  # Changing the max recursion to 10^6
+    img = Image.open('Images/ace2.JPG')
+
+    img2 = binary(img)  # Converting to a binary image, based on some given benchmarks for RGB values
```

Comment:
Naming different images according to what image they hold would make sense here. So img2 would be binaryImg.